packages/next/src/experimental/testmode/playwright/default-config.ts

```diff
@@ -0,0 +1,41 @@
+import { devices, type PlaywrightTestConfig } from '@playwright/test'
+
+/**
+ * This is the default configuration generated by Playwright as of v1.43.0 with some modifications.
+ *
+ * - the `testMatch` property is configured to match all `*.spec.js` or `*.spec.ts` files within the `app` and `pages` directories
+ * - the `use` property is configured with a baseURL matching the expected dev server endpoint (http://127.0.0.1:3000)
+ * - the `webserver` property is configured to run `next dev`.
+ */
+export const defaultPlaywrightConfig: PlaywrightTestConfig = {
+  testMatch: '{app,pages}/**/*.spec.{t,j}s',
+  fullyParallel: true,
+  forbidOnly: process.env.CI === 'true',
+  retries: process.env.CI === 'true' ? 2 : 0,
+  reporter: [['list'], ['html', { open: 'never' }]],
+  use: {
+    baseURL: 'http://127.0.0.1:3000',
+    trace: 'on-first-retry',
+  },
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
+    },
+
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+    },
+  ],
+  webServer: {
+    command: 'next dev',
```

Comment:
Let's make a note somewhere. This should depend on `process.env.CI` or on some command line arg. Normally, you'd locally debug tests using `next dev`, but on CI you'd do `next start`. This is important, because this kind of test should run as close to PROD-style as possible.

Reply:
fixed